Player/Player.tscn

```diff
@@ -1,7 +1,8 @@
-[gd_scene load_steps=5 format=3 uid="uid://dexjnwxb5r6uc"]
+[gd_scene load_steps=16 format=3 uid="uid://dexjnwxb5r6uc"]
 
 [ext_resource type="Script" path="res://Player/Player.gd" id="1_qs603"]
-[ext_resource type="Texture2D" uid="uid://bo5w3qxjpitl6" path="res://RawAssets/Character/Front/H0ndred_Front Idle_1.png" id="2_tivtx"]
+[ext_resource type="Texture2D" uid="uid://betlxtxfwfv01" path="res://RawAssets/FinalFinalFinal.png" id="2_xm4yt"]
```

Comment:
I changed the texture from a single frame (`H0ndred_Front Idle_1.png`), to a sprite atlas that came with the asset pack. I slightly modified it in Asesprite and re-exported it into `FinalFinalFinal.png`, but at this point I think I was just doing something wrong and the atlas would have worked fine on its own.